challenge-02/README.md

```diff
@@ -58,8 +83,29 @@ Crie uma função com as seguintes características:
 5. Se nenhum argumento for passado, retorne o valor booleano `false`.
 6. E ainda, se nenhuma das condições acima forem atendidas, retorne `null`.
 */
-?
+
+function somar( a, b , c ){
+
+       if( a === undefined && b === undefined && c === undefined ){
+           return false;
+        }
+        if( a !== undefined  && b === undefined  && c === undefined){
+            return a;
+        }
+        else if(c === undefined){
+            return a+b;
+        }
+        else if(c !== undefined){
+            return (a+b)/c;
+        }
+        else if(false){
```

Comment:
-1
Para essa função, nunca vai chegar a retornar `null`. Mas essa condição não diz muita coisa. Você está dizendo explicitamente que não vai entrar aqui, pois se o valor do `if` for false, ele passa direto. Como você está passando `false` explicitamente aqui, você diz para nunca entrar aqui de qualquer forma. Poderia usar só o `else` :)